tkg/managementcomponents/helper.go

```diff
@@ -129,6 +131,15 @@ func convertToString(config interface{}) string {
 	return ""
 }
 
+// convertToJsonString converts config into json string type, and return "" if config is not set
+func convertToJsonString(config interface{}) string {
+	if config != nil {
+		jsonStr, _ := json.Marshal(config.(string))
```

Comment:
1. We should not ignore `json.Marshal` error since it does have a chance that users input an invalid format.
2. It is not a safe way to convert interface to string using `config.(string)`
3. the variable jsonStr is actually a `byte[]` type, so you should not call it jsonStr, it is better to use jsonBytes.

Reply:
The whole part is better written like this: 

```go
if config != nil {
     jsonBytes, err := json.Marshal(fmt.Sprint(config))
     if err != nil {
        //  handle the error 
     }
      return string(jsonBytes)
}